projects/1.16/assets/hanami/hanami/lang/zh_cn.json

```
@@ -1 +1,77 @@
-{}
\ No newline at end of file
+{
+  "_comment": "樱木类",
+  "block.hanami.stripped_sakura_log": "条纹樱花原木",
+  "block.hanami.sakura_log": "樱花原木",
+  "block.hanami.stripped_sakura_wood": "条纹樱花木",
+  "block.hanami.sakura_wood": "樱花木",
+  "block.hanami.sakura_planks": "樱花木板",
+  "block.hanami.sakura_slab": "樱花木半砖",
+  "block.hanami.sakura_stairs": "樱花木楼梯",
+  "block.hanami.sakura_pressure_plate": "樱花木压力板",
+  "block.hanami.sakura_fence": "樱花木栅栏",
+  "block.hanami.sakura_fence_gate": "樱花木栅栏门",
+  "block.hanami.sakura_button": "樱花木按钮",
+  "block.hanami.sakura_trapdoor": "樱花木活板门",
+  "block.hanami.sakura_door": "樱花木门",
+  "block.hanami.sakura_sign": "樱花木告示牌",
+  "block.hanami.sakura_leaves": "樱花树叶",
+  "block.hanami.sakura_sapling": "樱花树苗",
```

```suggestion
  "block.hanami.sakura_leaves": "樱花树树叶",
  "block.hanami.sakura_sapling": "樱花树树苗",
```
同步原版叠词 https://minecraft.fandom.com/zh/wiki/%E6%A0%91%E8%8B%97